ng2-components/ng2-alfresco-documentlist/src/components/document-list.component.ts

```diff
@@ -697,7 +704,7 @@ export class DocumentListComponent implements OnInit, OnChanges, AfterContentIni
         }
     }
 
-    onNodeDblClick(node: MinimalNodeEntity) {
+    onNodeDblClick(node: any) {
```

Comment:
What is the purpose of removing the strong type and replacing with "any" in this case?

Reply:
There is no benefit in changing that, I will revert that, thank you!